vnext/ReactUWP/Views/ControlViewManager.cpp

```diff
@@ -65,6 +70,20 @@ void ControlViewManager::UpdateProperties(ShadowNodeBase *nodeToUpdate, const fo
   }
 
   Super::UpdateProperties(nodeToUpdate, reactDiffMap);
+
+  if (finalizeBorderRadius) {
+    if (control.try_as<winrt::Windows::UI::Xaml::Controls::IControl7>()) {
+      UpdateCornerRadiusOnElement(nodeToUpdate, control);
+    }
+  }
+}
+
+void ControlViewManager::InitializeDefaultProperties(XamlView view) {
+  // Set the default cornerRadius to 0 for Control: WinUI usually default cornerRadius to 2
+  if (auto control = view.try_as<winrt::Windows::UI::Xaml::Controls::IControl7>()) {
+    winrt::Windows::UI::Xaml::CornerRadius cornerRadius{0};
+    control.CornerRadius(cornerRadius);
```

Comment:
>control.CornerRadius(cornerRadius); [](start = 4, length = 35)

I didn't do the testing, maybe control.CornerRadius( {0} ); work

Reply:
Works!

---
In reply to: [359606312](https://github.com/microsoft/react-native-windows/pull/3793#discussion_r359606312) [](ancestors = 359606312)